### ISSUE TITLE
After filtering events in Search, Action bar title is incorrect

### ISSUE THREAD

**Issue by anhanh11001** [fossasia/open-event-android#1896]
**Describe the bug**
After filtering events in Search, Action bar title is incorrect

**Screenshots**
<img src="https://i.ibb.co/vxhJWnT/ezgif-2-6f02c7e62f71.gif" alt="ezgif-2-6f02c7e62f71" width="300">  <img src="https://i.ibb.co/c8KrRYZ/ezgif-2-74b99c2ef8a1.gif" alt="ezgif-2-74b99c2ef8a1" width="300">

**Would you like to work on the issue?**
Yes

**Comment by anhanh11001** [comment]
When I tick on the tick icon in FilterFragment (redirect to SearchResultsFragment) -> Here is the log from SearchResultsFragment and SearchFragment. It shows that SearchFragment still calls its method in the backstack... You have any ideas on this??

@liveHarshit 
```
2019-06-04 15:11:03.721 23936-23936/com.eventyay.attendee D/SearchResultsFragment: DEBUGGING onCreate SearchResultsFragment
2019-06-04 15:11:03.848 23936-23936/com.eventyay.attendee D/Utils: DEBUGGING TITLE: Search results
2019-06-04 15:11:03.852 23936-23936/com.eventyay.attendee D/SearchResultsFragment: DEBUGGING onCreateView SearchResultsFragment
2019-06-04 15:11:03.878 23936-23936/com.eventyay.attendee D/SearchFragment: DEBUGGING onCreateView SearchFragment
2019-06-04 15:11:03.901 23936-23936/com.eventyay.attendee D/Utils: DEBUGGING TITLE: Search
2019-06-04 15:11:03.905 23936-23936/com.eventyay.attendee D/SearchFragment: DEBUGGING onViewCreated SearchFragment
2019-06-04 15:11:03.907 23936-23936/com.eventyay.attendee D/SearchFragment: DEBUGGING onResume SearchFragment
2019-06-04 15:11:03.997 23936-23936/com.eventyay.attendee D/SearchResultsFragment: DEBUGGING onResume SearchResultFragment
```

**Comment by liveHarshit** [comment]
@anhanh11001 Search fragment is still running in the background. When we open search fragment, onCreate and onCreateView called. After redirecting to search results, onDestroyView is called, so after returning onCreateView is called not onCreate again. So the fragment is running in the back stack because of onDestroy of the fragment never calls.

**Comment by anhanh11001** [comment]
I need some helps here as I couldn't find way to solve this issue, I'm not sure if you understand the logs above. So when I make a filter from SearchFilterFragment (clicking tick icon), this action is taken
```
<action
            android:id="@+id/action_search_filter_to_search_results"
            app:popUpTo="@id/searchResultsFragment"
            app:popUpToInclusive="true"
            app:destination="@id/searchResultsFragment"/>
```
 here is what happens:

1. onCreated() of SearchResultFragment is called
2. onCreateView() of SearchResultFragment is called -> setting Action Bar title to "Search Results"
3. onCreate() of SearchFragment is called, even though SearchFragment is in the backstack.
4. onCreateView() of SearchFragment is called -> setting Action Bar title to "Search" // Bug here
...
5. onResume() of SearchResultFragment is called here

**Comment by liveHarshit** [comment]
> app:popUpToInclusive="true"

Set it to false. And use `SearchFilterFragmentDirection` to navigate.

**Comment by anhanh11001** [comment]
It is using `SearchFilterFragmentDirection` already. If popUpToInclusive is set to false, there would be 2 SearchResultFragment in the backstack

**Comment by liveHarshit** [comment]
> If popUpToInclusive is set to false, there would be 2 SearchResultFragment in the backstack

Have you tried?

**Comment by liveHarshit** [comment]
The same problem is fixed in PR #1922

**Comment by anhanh11001** [comment]
> Have you tried?

I have tried.
> The same problem is fixed in PR #1922

I'll take a look at that

**Comment by liveHarshit** [comment]
> 4\. onCreateView() of SearchFragment is called -> setting Action Bar title to "Search" // Bug here

Yes I also got the same problem while using `.popBackStack(destinationId)` but it was solved with using navigation fragment directions.
